src/components/StrategyEditor/StrategyEditor.js

```diff
@@ -394,7 +408,6 @@ export default class StrategyEditor extends React.PureComponent {
       createNewStrategyModalOpen,
       openExistingStrategyModalOpen,
     } = this.state
-
     if (!strategy) {
```

Comment:
Looks great!
Could you please also check if strategy is empty, so the page will display 'Create a new strategy or open an existing one.' modal:

```suggestion
if (!strategy || _isEmpty(strategy)) {
```